assets/blocks/course-categories-block/block.json

```diff
@@ -0,0 +1,63 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "sensei-lms/course-categories",
+	"category": "theme",
+	"textdomain": "sensei-lms",
+	"attributes": {
+		"align": {
+			"type": "string",
+			"default": "left"
+		},
+		"categoryStyle": {
+			"type": "object",
+			"default": {
+				"classes": [],
+				"style": {
+					"backgroundColor": "#3858E9",
```

Comment:
Ok! I am now loading the theme colors from here 6c2dff7df63f026e48dbf7109364518e7e533531 following some examples from the Gutenberg.

I know that we have this code https://github.com/Automattic/sensei/blob/6c2dff7df63f026e48dbf7109364518e7e533531/assets/react-hooks/probe-styles.js#L24

But I found that both approaches are reaching the same result, but I don't have sure if there is some special context that is not described in p2 post (I tried to find something related to colors).

Reply:
We chatted, and you were planning to connect with @renatho to learn a bit more about this and see if we could potentially eliminate color probing in the future.

Reply:
Hey there!

Also adding here the context of the conversation.

I'd love to eliminate the colors probe since it's a big hack. 😜 One approach we're using now in Interactive Video is to use an `<a>`, and inherit the colors from the theme [through `currentColor`](https://github.com/Automattic/sensei-pro/blob/5675c64a9df404544ecb6c8c2ebd359645033df7/modules/interactive-blocks/assets/interactive-video/pause-point-block/pause-point.scss#L31), but it can be a little limited depending on where we need to use the colors.

Also, just adding that I didn't explore it recently to see if there's a new and better approach from WordPress. 😉

Reply:
I updated (e31e62178633e4938d73009a47931a876ce3a570) the code to use the probes to support themes that are not defining  'primary' and 'background-colors' on the theme palette.